docs/docs/reference/metaprogramming/macros.md

```diff
@@ -40,7 +40,7 @@ def assertImpl(expr: Expr[Boolean])(using Quotes) = '{
 }
 
 def showExpr(expr: Expr[Boolean])(using Quotes): Expr[String] =
-  '{ "<some source code>" } // Better implementation later in this document
+  '{ [actual implementation later in this document] }
```

Comment:
```suggestion
  '{ /* actual implementation later in this document */ }
```

Reply:
That will also be eaten by whatever tool transforms the markdown to HTML, just as the other comment (probably all valid Scala comments?). I've no idea why a markdown-to-HTML converter would do that…

Reply:
Handling of angle brackets in code blocks is mangled by scala style at https://github.com/lampepfl/dotty/pull/12734

Reply:
Probably, it is very common for Markdown code comments to require special handling of angle brackets.  But I am talking about code comments, i.e. the `\\ text` and `\* text */` which simply disappear in the htmlized output.

Reply:
To confirm, if you remove `scala` from the triple-backticks, the angle brackets render correctly.
```
'{ "&lt;some source code&gt;" } // Better implementation later in this document
```

Reply:
Otherwise it xmlifies as
```
</span><span id="11" class="">  '{ "
        <some source code>
         " } 
         <span class="hideable">// Better implementation later in this document</span>

        </some></span><pre></pre></code></pre>
```
Apologies if this was already discussed elsewhere, just mentioning what I noticed this morning.

Reply:
The current version makes it readable. We can change it later if we want when the underlying issue is fixed.